frontend/src/app/bet/index.tsx

```diff
@@ -1,15 +1,41 @@
 import React from 'react';
-import { View } from 'react-native';
-import { globalStyles } from '../../styles/styles';
-import PremText from '../../components/basic/PremText';
+import { View, ScrollView, StyleSheet } from 'react-native';
+import { colors } from '../../styles/styles';
+import { MatchUp } from '../../components/bet/MatchUp';
+import { Gameweek } from '../../components/bet/Gameweek';
+import { Confirm } from '../../components/bet/Confirm';
 
 const Bet = () => {
   return (
-    <View style={globalStyles.container}>
-      <PremText>My Bet</PremText>
-      <PremText>Gameweek 1</PremText>
+    <View style={styles.container}>
+      <Gameweek />
+      <ScrollView>
+        <MatchUp />
+        <MatchUp />
+        <MatchUp />
+        <MatchUp />
+        <MatchUp />
+        <MatchUp />
+        <MatchUp />
+      </ScrollView>
+      <Confirm />
     </View>
   );
 };
 
+const styles = StyleSheet.create({
+  container: {
```

Comment:
We have a container style item inside of globalStyles to maintain consistency. We should rather use that and update if needed.

Reply:
Fixed